issue98-websockets/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "issue98-websockets",
+  "version": "1.0.0",
+  "description": "Building a chat using WebSockets",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node src/server.js",
+    "dev": "nodemon src/server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/picolloo/CurriculumExpansion.git"
+  },
+  "author": "Lucas Picollo",
```

Comment:
I'm not sure if we want to leave the rights with @picolloo or @freeCodeCamp would take over it.

cc: @Ethan-Arrowood